ui/src/main/js/page/dashboard/index.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/modernmaster/forecaster/pullRequest?prid=7238741)
```suggestion
"use strict";
```